tox.ini

```diff
@@ -189,6 +189,7 @@ plugins = coverage_pyver_pragma
 
 [coverage:report]
 fail_under = 100
+show_missing = true
```

Comment:
I was having trouble figuring out which lines were missing coverage, given the requirement that it be 100% coverage.